Sources/Purchasing/StoreKit1/StoreKitWrapper.swift

```diff
@@ -34,6 +34,14 @@ class StoreKitWrapper: NSObject, SKPaymentTransactionObserver {
     @available(iOS 8.0, macOS 10.14, watchOS 6.2, macCatalyst 13.0, *)
     static var simulatesAskToBuyInSandbox = false
 
+    var currentStorefront: String? {
+        guard #available(iOS 13.0, tvOS 13.0, macOS 10.15, watchOS 6.2, *) else {
+            return nil
+        }
+
+        return self.paymentQueue.storefront?.countryCode
```

Comment:
Maybe we could create our own `Storefront` that abstracts `SKStorefront` and `StoreKit.Storefront` so we can pass that around instead of a String.

Reply:
Yeah, that's a good idea

Reply:
#1500